src/components/NewEventDialog.js

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback } from 'react';
+import { useMutation } from '@apollo/react-hooks';
+import useForm from 'react-hook-form';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { TextField, Button, Grid, Box } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import useStyle from './UseStyle';
+
+const { DateTime } = require('luxon');
```

Comment:
This should work with the ES6 style too.
```suggestion
import { DateTime } from 'luxon';
```

Reply:
Done